README.md

```diff
@@ -45,9 +45,13 @@ or if you have any trouble with this, please try the new standalone install step
 This might require elevated privileges in some environments.
 
 ```sh
-$ npm install --global --production resin-cli
+$ npm install --global --production resin-cli --unsafe-perm
```

Comment:
Some could argue to have this as a note "if you get permission errors", in an attempt to make the default suggested installation command less intimidating. But I'm fine with this as is as well :)

Reply:
Yeah, I wasn't sure about that... I think I'd rather have it here by default though. From the reports and Zack's experience at the hackathon recently, it sounds like this is required for large number of users now, and nobody ever reads the fine print :smiley:.